### ISSUE TITLE
Can't compile with npm

### ISSUE THREAD

**Issue by andrescirulo** [vaadin/gwt-polymer-elements#131]
I'm getting an error when i run the npm install command.
It creates two Java interfaces with weird name:
- TransitionendEventparamnodeObjectdetailContainstheanimatednode.Event
- TransitionendEventparamnodeObjectdetailContainstheanimatednode.EventHandler
  So when it tries to compile it breaks.
  If i'm not mistaken the problem have to do with the paper-button.html that has this at the end

```
/**
Fired when the animation finishes.
This is useful if you want to wait until
the ripple animation finishes to perform some action.

@event transitionend
      Event param: {{node: Object}} detail Contains the animated node.
*/
```

Any suggestion to how to solve this?
I attach the console output just in case.

Thanks

[out.txt](https://github.com/vaadin/gwt-polymer-elements/files/401613/out.txt)

**Comment by ghost** [comment]
see here: [gwt-api-generator/issues/65](https://github.com/vaadin/gwt-api-generator/issues/65)

**Comment by manolo** [comment]
Fixed in gwt-api-generator/ecb419c8c3264bab7e93c54ac7f6c6792bc4a1e6
